### ISSUE TITLE
Latest snapshots and doors

### ISSUE THREAD

**Issue by nickbe** [nuvie/nuvie#67]
It seems that doors which are facing up are not openable anymore in the latest builds.

**Comment by malignantmanor** [comment]
I'm not able to repeat this bug. Are you still having issues with the latest snapshot?

**Comment by nickbe** [comment]
I encountered this with several doors and from time to time. Let me see if I can reproduce this.

**Comment by shevegen** [comment]
Any luck finding out which doors are naughty?

**Comment by nickbe** [comment]
Yep, just found it again. The attached screenshot shows the problem. If I look at the door or try to open it all I get in response is "Thou dost see earth" and the door simply doesn't seem to exist. I also attached the savegame. Hope you're able to find what the problem is here.

[nuvieU6005.zip](https://github.com/nuvie/nuvie/files/1353185/nuvieU6005.zip)

![screenshot_49](https://user-images.githubusercontent.com/10890765/31137586-fe130de8-a86b-11e7-8f8f-1ed478b2c0ae.png)

**Comment by malignantmanor** [comment]
Thanks, there is a throwing axe on the same tile as the closed door. If you move far enough away from that it is permanently removed from the game, the door works fine. Maybe Eric can figure out how to fix this issue. Closed doors should take priority over anything else on the same tile.

**Comment by Zabeus** [comment]
This is an interesting savestate. I wonder if the axe was dropped there and then an NPC on a schedule went through the door and closed it. The player shouldn't be able to close the door if an object is in the way. (in original U6 it says "You can't do that now!" and in Nuvie the object would be selected instead)

In the original you would be able to select the tile left of the door and open it, but for some reason in Nuvie you can't. It should at least say you can't use the throwing axe, and not just "Use - nothing".

Maybe we just need to prioritize permanent world objects over temp objects.

**Comment by nickbe** [comment]
Well, I'm pretty sure that I didn't drop an axe here. So maybe there's a bug when saving the game itself.
